Marlin/src/Marlin.cpp

```diff
@@ -218,15 +218,40 @@ void setup_killpin() {
   #endif
 }
 
-#if ENABLED(FILAMENT_RUNOUT_SENSOR)
+if ENABLED(FILAMENT_RUNOUT_MULTI_SENSORS)
```

Comment:
because of the renaming , i don't have merge all units , it's why , you have to make it yourself , for all units